src/main/java/com/sample/project/demo/entity/Tasks.java

```diff
@@ -0,0 +1,27 @@
+package com.sample.project.demo.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.time.ZonedDateTime;
+import java.util.List;
+
+@Data
```

Comment:
@Data is only required, you can remove @AllArgsConstructor
@NoArgsConstructor
@ToString